client/extensions/woocommerce/app/dashboard/setup.js

```diff
@@ -0,0 +1,196 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import SetupFooter from './setup-footer';
+import SetupHeader from './setup-header';
+import SetupTask from './setup-task';
+
+class Setup extends Component {
+	static propTypes = {
+		onFinished: PropTypes.func,
+		site: React.PropTypes.shape( {
```

Comment:
Both this line and the `slug` line below can just use `PropTypes` instead of `React.PropTypes`.